tests/Gating.UnitTests/GateContextUnitTests.cs

```diff
@@ -21,6 +21,11 @@ namespace Microsoft.Omex.Gating.UnitTests
 	/// </summary>
 	public sealed class GateContextUnitTests : UnitTestBase
 	{
+#pragma warning disable SM02239 // hard coded e-mail for unit tests
+		private static readonly string s_userIdentifierInGroup = "test1@microsoft.com";
```

Comment:
These can be made `const`, which is preferable for `string`s as they're more efficient than `static readonly`

Reply:
Good point, thanks, replaced